docs/testing/debug-desktop-using-edge-chromium.md

```diff
@@ -33,18 +34,18 @@ These instructions assume you have experience using the command line, understand
 
    - If you want to create a project to experiment with debugging in Visual Studio Code, use the [Yeoman generator for Office Add-ins](../develop/yeoman-generator-overview.md). Use any one of our quick start guides, such as the [Outlook add-in quick start](../quickstarts/outlook-quickstart.md), in order to do this.
    - If you want to debug an existing project that was created with Yo Office, skip to the next step.
-   - If you want to debug an existing project that was not created with Yo Office, complete the procedure in the [Appendix A](#appendix-a) and then return to the next step of this procedure.
+   - If you want to debug an existing project that was not created with Yo Office, complete the procedure in the [Debug a project not created with yo office](#debug-a-project-not-created-with-yo-office) and then return to the next step of this procedure.
 
 1. Open VS Code and open your project in it.
 
-1. Choose  **View** > **Run** or enter **Ctrl+Shift+D** to switch to debug view.
+1. Choose **View** > **Run** or enter **Ctrl+Shift+D** to switch to debug view.
 
 1. From the **RUN AND DEBUG** options, choose the Edge Chromium option for your host application, such as **Outlook Desktop (Edge Chromium)**. Select **F5** or choose **Run** > **Start Debugging** from the menu to begin debugging. This action automatically launches a local server in a Node window to host your add-in and then automatically opens the host application, such as Excel or Word. This may take several seconds.
 
    > [!TIP]
    > If you aren't using a project created with Yo Office, you may be prompted to adjust a registry key. While in the root folder of your project, run the following in the command line.
    >
-   > ``` command&nbsp;line
```

Comment:
Do we no longer have to write command line this way? When did that change?

Reply:
Hmmm, not sure.

Reply:
If it renders fine, great! Maybe double check after this gets merged. If all is well, we should change that everywhere.